images/build/debian-iptables/distroless/Dockerfile

```diff
@@ -0,0 +1,54 @@
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG GORUNNERIMAGE
+# Using directly debian base, as we are going to just copy the required binaries
+FROM debian:bullseye as build
+
+COPY stage-binary-and-deps.sh /
```

Comment:
@BenTheElder any better other way here instead of copying your whole script? I think KinD deserves the credit here!

Reply:
I'm not sure what the best way is, sometimes I've used git `Co-authored-by`  https://github.com/kubernetes-sigs/oci-proxy/pull/5/commits/340c13dc01a9bd585cc5d54780634c67d0513ce3

Reply:
ok cool, I've added a specific commit just for this script and added you as a co-author in the commit message